app/src/main/java/com/besome/sketch/editor/view/ViewEditor.java

```diff
@@ -337,7 +337,6 @@ public boolean onTouch(View view, MotionEvent motionEvent) {
                     boolean z = false;
                     for (int i3 = 0; i3 < uyVar.getData().size(); i3++) {
                         ViewBean viewBean = uyVar.getData().get(i3);
-                        if (c(viewBean)) {
```

Comment:
@PranavPurwar explanation?

Reply:
look at the rest of the commit, the entire method used to do nothing except return `true`